jsonnet/dashboards/apps/loki.libsonnet

```diff
@@ -0,0 +1,246 @@
+        .addPanel(msgs_req_graph { tooltip+: { sort: 2 } }, { x: 0, y: sy+8, w: 12, h: 7 })
+        .addPanel(msgs_resp_graph { tooltip+: { sort: 2 } }, { x: 12, y: sy+8, w: 12, h: 7 })
+        ;
+   local panel_ingester(sy) = row.new('Ingester', collapse=true) { gridPos: { x: 0, y: sy, w: 24, h: 1 } }
+        .addPanel(ingester_blocks { tooltip+: { sort: 2 } }, { x: 0, y: sy+1, w: 8, h: 7 })
+        .addPanel(ingester_chunk_size { tooltip+: { sort: 2 } }, { x: 8, y:sy+1, w: 8, h: 7 })
+        .addPanel(ingester_chunk_age { tooltip+: { sort: 2 } }, { x: 16, y: sy+1, w: 8, h: 7 })
+        .addPanel(ingester_chunk_cmpratio { tooltip+: { sort: 2 } }, { x: 0, y: sy+8, w: 8, h: 7 })
+        .addPanel(ingester_chunk_enctime { tooltip+: { sort: 2 } }, { x: 8, y: sy+8, w: 8, h: 7 })
+        .addPanel(ingester_lines { tooltip+: { sort: 2 } }, { x: 16, y: sy+8, w: 8, h: 7 })
+        ;
+   local panel_cache(sy) = row.new('Cache',collapse=true) {gridPos:{x: 0, y:sy, w:24, h:1}}
+        .addPanel(cache_size { tooltip+: { sort: 2 } }, {x: 0, y: sy+1, w: 12, h: 7})
+        .addPanel(cache_fetched_keys { tooltip+: { sort: 2 } }, {x: 12, y: sy+1, w: 12, h: 7})
+        .addPanel(cache_hits_keys { tooltip+: { sort: 2 } }, {x: 0, y: sy+8, w: 12, h: 7})
+        ;
+   local panel_index(sy) = row.new('Index',collapse=true) {gridPos:{x: 0, y:sy, w:24, h:1}}
+        .addPanel(chunk_store_chunks { tooltip+: { sort: 2 } }, {x: 0, y: sy+1, w: 8, h: 7})
+        .addPanel(chunk_store_entries { tooltip+: { sort: 2 } }, {x: 8, y: sy+1, w: 8, h: 7})
+        .addPanel(index_lookups { tooltip+: { sort: 2 } }, {x: 16, y: sy+1, w: 8, h: 7})
+        .addPanel(series_pre_intersection { tooltip+: { sort: 2 } }, {x: 0, y: sy+8, w: 12, h: 7})
+        .addPanel(series_post_intersection { tooltip+: { sort: 2 } }, {x: 12, y: sy+8, w: 12, h: 7})
+        ;
+   local panel_querrier(sy) = row.new('Querier',collapse=true) {gridPos:{x: 0, y:sy, w:24, h:1}}
+        .addPanel(querier_cache_hits { tooltip+: { sort: 2 } }, {x: 0, y: sy+1, w: 8, h: 7})
+        .addPanel(querier_cache_puts { tooltip+: { sort: 2 } }, {x: 8, y: sy+1, w: 8, h: 7})
+        .addPanel(querier_cache_gets { tooltip+: { sort: 2 } }, {x: 16, y: sy+1, w: 8, h: 7})
+        .addPanel(querier_cache_corruptions { tooltip+: { sort: 2 } }, {x: 0, y: sy+8, w: 12, h: 7})
+        .addPanel(querier_cache_errors { tooltip+: { sort: 2 } }, {x: 12, y: sy+8, w: 12, h: 7})
+        ;
+   local panels = [
+        /*Overview*/
+        row.new('Overview') { gridPos: { x: 0, y: 0, w: 24, h: 1 } },
+        version {gridPos: { x: 0, y: 1, w: 4, h: 5 }},
+        msgs { gridPos: { x: 4, y: 1, w: 4, h: 5 } },
+        errors { gridPos: { x: 8, y: 1, w: 4, h: 5 } },
+        panic { gridPos: { x: 12, y: 1, w: 4, h: 5 } },
+        /*Other Panels*/
+        panel_msgs(6),panel_api(14),panel_cache(29),panel_querrier(44),panel_index(59),panel_ingester(74)];
+
+   dashboard.new(
+        'Loki',
```

Comment:
Is it dashboard for loki or loki-distributed or for both?

Reply:
If you decided to go with loki-distributed you should also rename file to `loki-distributed.libsonnet`